src/main/java/com/squareup/kotlinpoet/CodeBlock.kt

```diff
@@ -138,6 +138,10 @@ class CodeBlock private constructor(
     internal val formatParts = mutableListOf<String>()
     internal val args = mutableListOf<Any?>()
 
+    fun isEmpty() = formatParts.isEmpty()
+
+    fun isNotEmpty() = !isEmpty()
```

Comment:
:+1: to these two methods. I'm pretty sure there's a few places in the codebase where they can be used.